.editorconfig

```diff
@@ -0,0 +1,21 @@
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
```

Comment:
I agree with the sentiment, but I feel a bit iffy about such a wild-reaching wildcard.

Would it cause fewer or more issues to restrict this to `[*.{js,ts,jsx,tsx,sh}]`?

(I really don't know, so I leave this at your discretion 🙂)

Reply:
Hmm good point. As far as I know <kbd>LF</kbd> instead of <kbd>CR</kbd> <kbd>LF</kbd> doesn't cause any issues on Windows and `notepad.exe` supports them now. Some tools choke on the latter, but I'm not aware of tools that still choke on the former.

I'd say it's safer to rely on <kbd>LF</kbd> and possibly add exceptions on a case-by-case basis (I don't think that they will be needed though).

Let me know your thoughts!